spec/schemas/flow_framework.errors.yaml

```diff
@@ -66,6 +66,35 @@ components:
               example: Workflow ID can not be null
             status:
               type: integer
+    DuplicateKeyError:
+      type: object
+      properties:
+        error:
+          type: string
+          description: Error message when a duplicate key is found in the request.
```

Comment:
Not sure "duplicate key" is self-descriptive here.  Is this a duplicate param, or a duplicate field in the body JSON?

Reply:
This looks correct to me, should match the code.

Reply:
For now as @dblock says it's ok to match the code. The error message should probably be "duplicate param" but not a big deal to leave it as is.  If you get the message you'll know what you did wrong.   [Whoever wrote that code](https://github.com/opensearch-project/flow-framework/pull/525) should have thought better. ;)